public/sample-bulletin.json

```diff
@@ -0,0 +1,43 @@
+{
+  "forecastdate": "25-09-2024 13:53",
```

Comment:
what url did you just to retrieve this?  As `mode` did you specify `all` .e.g

```
"https://london-airtext-forecasts-api-gateway-7x54d7qf.nw.gateway.dev/getforecast/all? 
```

When I make request with the "all" mode ( to return both pollutant and non-pollutant data) I do get the pollen, UV and temp info:

```
curl "https://london-airtext-forecasts-api-gateway-7x54d7qf.nw.gateway.dev/getforecast/all?numdays=3&zone=Haringey&key=SECRET"
```

gives:

```json

{
  "forecastdate": "30-09-2024 09:08",
  "timestamp": 1727687321964.663,
  "zones": [
    {
      "forecasts": [
        {
          "NO2": 1,
          "O3": 2,
          "PM10": 1,
          "PM2.5": 1,
          "forecast_date": "2024-09-30",
          "non_pollution_version": null,
          "pollen": 0,
          "pollution_version": 202409300807,
          "rain_am": 11.76,
          "rain_pm": 0.26,
          "temp_max": 16.3,
          "temp_min": 10.9,
          "total": 2,
          "total_status": "LOW",
          "uv": 1,
          "wind_am": 5.6,
          "wind_pm": 7.6
        },
        {
          "NO2": 1,
          "O3": 2,
          "PM10": 1,
          "PM2.5": 1,
          "forecast_date": "2024-10-01",
          "non_pollution_version": null,
          "pollen": 0,
          "pollution_version": 202409300259,
          "rain_am": 1.3,
          "rain_pm": 0.62,
          "temp_max": 15.4,
          "temp_min": 11.0,
          "total": 2,
          "total_status": "LOW",
          "uv": 2,
          "wind_am": 6.0,
          "wind_pm": 5.7
        },
        {
          "NO2": 1,
          "O3": 2,
          "PM10": 1,
          "PM2.5": 1,
          "forecast_date": "2024-10-02",
          "non_pollution_version": null,
          "pollen": 0,
          "pollution_version": 202409300259,
          "rain_am": 0.87,
          "rain_pm": 0.21,
          "temp_max": 16.3,
          "temp_min": 9.7,
          "total": 2,
          "total_status": "LOW",
          "uv": 2,
          "wind_am": 5.0,
          "wind_pm": 5.5
        }
      ],
      "zone_id": 14,
      "zone_name": "Haringey",
      "zone_type": 1
    }
  ]
}
```

Reply:
thanks Ed, yes I can see I wasn't specifying `all`. rectifying now...